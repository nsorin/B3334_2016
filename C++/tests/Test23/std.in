LOAD t1
HIT rect1 20 20
HIT rect1 200 200
HIT reu1 20 20
HIT reu1 200 200
HIT r1 7 7
HIT r1 3 3
CLEAR
LOAD t2
HIT r1 3 3
EXIT
