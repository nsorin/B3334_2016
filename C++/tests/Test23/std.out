#LOAD t1
OK
#HIT rect1 20 20
YES
#HIT rect1 200 200
NO
#HIT reu1 20 20
YES
#HIT reu1 200 200
NO
#HIT r1 7 7
YES
#HIT r1 3 3
NO
#CLEAR
OK
#LOAD t2
OK
#HIT r1 3 3
YES
#EXIT
